includes/Modules/Ads.php

```diff
@@ -131,12 +131,13 @@ protected function setup_assets() {
 				$assets[] = new Script(
 					'googlesitekit-ads-pax-integrator',
 					array(
-						'src'          => 'https://www.gstatic.com/pax/dev/pax_integrator.js',
```

Comment:
I saw the note in the description – the issue here is about the added query param when the version isn't `null`, but the same applies to `latest`, so let's keep it as it was, since it should already be fixed via the added `version` below.

Reply:
It wasn't loading at all for me with the previous code, did the Ads team recently add support for the `version` query string?

Reply:
I misread this comment, but I see now, we can leave `dev` instead of `latest`.